table/server/underdb/glue/src/main/java/alluxio/table/under/glue/GlueDatabase.java

```diff
@@ -386,6 +386,11 @@ public UdbTable getTable(String tableName) throws IOException {
               .withTableName(tableName);
       if (glueClient.getPartitions(getPartitionsRequest).getPartitions() != null) {
         partitions = glueClient.getPartitions(getPartitionsRequest).getPartitions();
+        LOG.debug("Get {} partitions from Table {} Database {}.",
```

Comment:
It would be helpful for this message to be distinct from other messages, so this could something like "Glue table {}.{} has {} partitions."

Reply:
Changed.